.github/workflows/deploy-html-storybook-prod.yml

```diff
@@ -82,8 +82,6 @@ jobs:
     name: Deploy to Amazon S3
     runs-on: ubuntu-latest
     needs: build
-    # Run if html is affected or manually triggered
-    if: ${{ needs.determine-affected.outputs.html_affected == 'true' || github.event_name == 'workflow_dispatch' }}
```

Comment:
Dont we need this here for manual deployment? Happy to fix it for you.

Reply:
It depends on `build`, which has the same reference.
Plus is doesn't reference `determine-affected` so it wouldn't work anyway.